package.json

```diff
@@ -11,8 +11,7 @@
     "url": "https://github.com/open-metadata/OpenMetadata.git"
   },
   "devDependencies": {
-    "quicktype": "^23.0.170",
-    "node-gyp": "^10.2.0"
```

Comment:
@chirag-madlani , don't we require node-gyp anymore?

Reply:
No everything is working fine without it.